SwiftUIDZ/SwiftUIDZ/ProductDetailView.swift

```diff
@@ -0,0 +1,161 @@
+// ProductDetailView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Детальное описание продукта
+struct ProductDetailView: View {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let yellowColorName = "pastelYellow"
+        static let purpleColorName = "pastelPurple"
+        static let darkPurpleColorName = "pastelDarkPurple"
+        static let redColorName = "pastelRed"
+        static let darkRedColorName = "pastelDarkRed"
+        static let navBarHeight: CGFloat = 30
+        static let twentyPadding: CGFloat = 20
+        static let cornerRadius: CGFloat = 8
+        static let bedImageName = "bed.double"
+        static let bedText = "Bed"
+        static let priceText = "$ 55"
+        static let heartIconImageName = "suit.heart"
+        static let buyNowButtonTitle = "Buy now"
+        static let bagIconImageName = "bag"
+        static let descriptionText = "Description"
+        static let charsText = "Chars:"
+        static let totalCharsText = "/ 150"
+    }
+
+    // MARK: - Body
```

Comment:
```suggestion
    // MARK: - Public properties
```
я так пишу, но если у тебя боди принимают - оставляй, я больше не буду обращать на это внимание